starfive/visionfive/v2/default.nix

```diff
@@ -1,15 +1,10 @@
 { config, lib, pkgs, ... }: {
   boot = {
-    # Force no ZFS (from nixos/modules/profiles/base.nix) until updated to kernel 6.0
-    supportedFilesystems =
-      lib.mkForce [ "btrfs" "reiserfs" "vfat" "f2fs" "xfs" "ntfs" "cifs" ];
     consoleLogLevel = lib.mkDefault 7;
-    kernelPackages = lib.mkDefault (pkgs.callPackage ./linux-6.6.nix {
-      inherit (config.boot) kernelPatches;
-    });
+    # Require at least Linux >= 6.9
+    kernelPackages = lib.mkDefault pkgs.linuxPackages_latest;
```

Comment:
Instead of a comment, you could add

```
  assertions = [
    {
      assertion = (lib.versionAtLeast config.boot.kernelPackages.kernel.version "6.9");
      message = "The Starfive Visionfive 2 requires a newer kernel version (>=6.9). Please upgrade nixpkgs for this system.";
    }
  ];
```
similar to https://github.com/NixOS/nixos-hardware/blob/057a7996d012f342a38a26261ee529cebb1755ef/raspberry-pi/5/default.nix#L29C1-L34C5